scripts/create-secret.sh

```diff
@@ -1,100 +1,87 @@
 #!/bin/bash
-set -euo pipefail
+set -e
 
-#---------------------------------------------------------------------------------------------------------------#
-#--- Script for creating secrets (SecureString) in the AWS parameter store using the path /k8s/[secret-name]----#
-#---------------------------------------------------------------------------------------------------------------#
+export AWS_PAGER=""
```

Comment:
unused?

Reply:
yes, disable the output from going to the pager

Reply:
so, no... its used 😄

Reply:
yep got it :)